Dockerfile

```diff
@@ -0,0 +1,35 @@
+# A container that launches the Polymesh node in unsafe development mode with an open WS port 9944.
+#
+# Build with `docker build . -t polymesh`
+
+FROM rust:latest
```

Comment:
it's a good idea to pin this to a version - `latest` is similar to `HEAD` ; building this container later could result in significant differences for anyone who hadn't already pulled a similar version

Reply:
https://hub.docker.com/_/rust/?tab=tags something like `1.43.1-slim-buster` would be good

Reply:
slim being the "slimified" version of debian ; see: https://hub.docker.com/_/debian `Image Variants` / https://github.com/debuerreotype/debuerreotype/blob/master/scripts/debuerreotype-slimify 

buster being debian buster (10)